kubernetes/kites/net-test-dev_ds.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: net-test-dev-ds
+  labels:
+    app: net-test-dev-ds
+spec:
+  selector:
+    matchLabels:
+      app: net-test-dev
+  template:
+    metadata:
+      labels:
+        app: net-test-dev
+    spec:
+      containers:
+        - name: net-test-dev-server
+          image: kitesproject/net-test-dev:1.0
```

Comment:
This should be called kitesproject/iperf3-....

Reply:
@ferrarimarco I had asked to change from iperf3 for the reasons explained above